generated/nidaqmx/task.py

```diff
@@ -1220,16 +1223,18 @@ class _TaskAlternateConstructor(Task):
     # Setting __slots__ avoids TypeError: __class__ assignment: 'Base' object layout differs from 'Derived'.
     __slots__ = []
 
-    def __init__(self, task_handle, interpreter):
+    def __init__(self, task_handle, interpreter, is_new_session_initialized=False):
         """
         Args:
             task_handle: Specifies the task handle from which to create a
                 Task object.
             interpreter: Specifies the interpreter instance.
+            is_new_session_initialized: Specifies if new session is initialized or not.
             
         """
         self._handle = task_handle
         self._interpreter = interpreter
+        self._is_new_session_initialized = is_new_session_initialized
```

Comment:
I think `_close_on_exit` is better

Reply:
Thanks for the information! Updated the name to _close_on_exit.